.github/workflows/python-validate.yml

```diff
@@ -77,3 +93,31 @@ jobs:
 
           python3 -m pip install pytest
           python3 -m pytest
+
+      - name: Install SDK Schema
+        if: steps.pathChanges.outputs.sdkSchema == 'true'
+        run: |
+          cd python/packages/sdk-schema
+
+          python3 -m pip install poethepoet
+          python3 -m poethepoet install
+
+      - name: Run SDK Schema unit tests
```

Comment:
I believe this should be run after the build.

Also, let's ensure that PR's just address the scope of the task. I see this change being proposed in two different PR's which should not be the case.

Either it should be proposed separately or included just in one where SDK Schema build is configured

Reply:
I moved `Run SDK Schema unit tests` after the build, but were you also referencing `Install SDK project and dependencies` as well?

And will do, I think this might have been the result of rebasing branch with `sc-25*`, I'll avoid doing that before they're merged with `main`.